cmd/crictl/exec.go

```diff
@@ -173,6 +173,14 @@ func stream(in, tty bool, url *url.URL) error {
 	logrus.Debugf("StreamOptions: %v", streamOptions)
 	if !tty {
 		return executor.Stream(streamOptions)
+	} else {
+		var detachKeys []byte
+		detachKeys, err = mobyterm.ToBytes("ctrl-p,ctrl-q")
```

Comment:
nit: could this be a const named variable so it's easier to reference

Reply:
Just clarifying, do you mean change `detchKeys` to a const or make the key sequence string (`"ctrl-p,ctrl-q"`) a const?

Reply:
the sequence string please :)

Reply:
Done